components/script/script_thread.rs

```diff
@@ -2916,66 +2926,91 @@ impl ScriptThread {
     /// Handles firing of transition-related events.
     ///
     /// TODO(mrobinson): Add support for more events.
-    fn handle_transition_event(
+    fn handle_transition_or_animation_even(
```

Comment:
event

Reply:
Oof. Thanks for pointing this out. :sweat_smile: